svgdigitizer/entrypoint.py

```diff
@@ -10,8 +10,9 @@
     Options:
       --help  Show this message and exit.
     Commands:
-      cv        Digitize a cylic voltammogram.
+      cv        Digitize a cylic voltammogram and create a frictionless...
       digitize  Digitize a plot.
+      figure    Digitize a figure with units on the axis and create a...
```

Comment:
Why the `...`?

Reply:
doctest creates a cutoff at 80 characters and the line above is the output from doctest.

Somehow `+NORMALIZED_WHITESPACE` does not work in this case.

Reply:
see #187